.packit.yml

```
@@ -59,3 +59,22 @@ jobs:
     dist_git_branches:
       - fedora-development
       - fedora-stable
+- job: propose_downstream
+  trigger: release
+  actions:
+    post-upstream-clone:
+        - "wget https://src.fedoraproject.org/rpms/bpfman/raw/main/f/bpfman.spec -O bpfman.spec"
+  specfile_path: bpfman.spec
+  dist_git_branches:
+    - fedora-all
+  upstream_tag_template: v{version}
```

I see there is a YAML syntax error.... We should move the job inside `jobs` list... Something like this should work:


```suggestion
  - job: propose_downstream
    trigger: release
    actions:
      post-upstream-clone:
          - "wget https://src.fedoraproject.org/rpms/bpfman/raw/main/f/bpfman.spec -O bpfman.spec"
    specfile_path: bpfman.spec
    dist_git_branches:
      - fedora-all
    upstream_tag_template: v{version}
```

---

Same for the other two...